content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -53,5 +53,19 @@ Datasets, metrics, and hyperparameters can be added to a project in two ways:
    file (within your Git repo) which contains your saved metrics. Refer to the
    section [on project settings] to learn how to specify the custom files.
 
+## Prepare Your Repositories to Run New Experiments
```

Comment:
💅🏼 Should we add another H2 earlier in this doc for the previous half? Otherwise the only entry in the right-side nav (ToC) is this header. See https://dvc-org-cml-in-studio-bgeulj3d.herokuapp.com/doc/studio/user-guide/prepare-your-repositories

Reply:
Added heading for `Set up datasets, metrics, and hyperparameters`.

Reply:
Nice, thanks.